scripts/benchmark-b1.py

```
@@ -100,7 +100,7 @@ def run_checked(
     input_text: str | None = None,
 ) -> str:
     """Run a command and return stdout, raising CaptureError on failure."""
-    result = subprocess.run(
+    result = subprocess.run(  # nosec -- nosemgrep: python.lang.security.audit.dangerous-subprocess-use-audit,python.lang.security.audit.dangerous-subprocess-use-tainted-env-args -- argv is a list (no shell); the inherited/derived environment configures the child process (as env=), it is never interpolated into a shell string.
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)